deployment/gcp/infrastructure/production/README.md

```diff
@@ -0,0 +1,30 @@
+# Instructions for production deployment
+
+0. Set up terraform backend as per `tf_backend/README.md`
+1. Update `production.auto.tfvars` file
+2. Initialize with `terraform init -backend-config=backend.conf`
+3. `terraform apply`
+
+   You will be prompted to enter your GCP project_id and service_account_email.
+
+   Alternatively, you can export the following environment variables so that these
+   variables are applied for each `terraform` command:
+
+   ```shell
+   export TF_VAR_gcp_project_id=<PROJECT_ID>
```

Comment:
@suzinyou These exported env variables did not stop my being prompted to enter them